src/test/lib/expression/pqp_subquery_expression_test.cpp

```diff
@@ -22,94 +23,124 @@ namespace opossum {
 class PQPSubqueryExpressionTest : public BaseTest {
  public:
   void SetUp() override {
-    table_a = load_table("resources/test_data/tbl/int_float.tbl");
-    Hyrise::get().storage_manager.add_table("int_float", table_a);
-    a_a = PQPColumnExpression::from_table(*table_a, "a");
-    a_b = PQPColumnExpression::from_table(*table_a, "b");
-
-    // Build a Subquery returning a SINGLE NON-NULLABLE VALUE and taking ONE PARAMETER
-    const auto parameter_a = placeholder_(ParameterID{2});
-    const auto get_table_a = std::make_shared<GetTable>("int_float");
-    const auto projection_a = std::make_shared<Projection>(get_table_a, expression_vector(add_(a_a, parameter_a)));
-    const auto limit_a = std::make_shared<Limit>(projection_a, value_(1));
-    pqp_single_value_one_parameter = limit_a;
-    parameters_a = {std::make_pair(ParameterID{2}, ColumnID{3})};
-    subquery_single_value_one_parameter =
-        std::make_shared<PQPSubqueryExpression>(pqp_single_value_one_parameter, DataType::Int, false, parameters_a);
-
-    // Build a Subquery returning a TABLE and taking NO PARAMETERS
-    const auto get_table_b = std::make_shared<GetTable>("int_float");
-    const auto table_scan_b = std::make_shared<TableScan>(get_table_b, greater_than_(a_a, 5));
-    pqp_table = table_scan_b;
-    subquery_table = std::make_shared<PQPSubqueryExpression>(pqp_table);
+    _table = load_table("resources/test_data/tbl/int_float.tbl");
+    Hyrise::get().storage_manager.add_table("int_float", _table);
+    _a = PQPColumnExpression::from_table(*_table, "a");
+    _b = PQPColumnExpression::from_table(*_table, "b");
+
+    // Create PQPSubqueryExpression
+    //  Build a PQP WITHOUT PARAMETERS
+    const auto get_table1 = std::make_shared<GetTable>(_table_name);
+    _pqp = std::make_shared<TableScan>(get_table1, greater_than_(_a, 5));
+
+    _pqp_subquery_expression = std::make_shared<PQPSubqueryExpression>(_pqp);
+
+    // Create PQPSubqueryExpression with Parameter
+    //  Build a PQP with ONE PARAMETER returning a SINGLE NON-NULLABLE VALUE
```

Comment:
The comments here look somewhat weird. Not (entirely) your fault - why do we (continue to) use caps here? Also, how does this line relate to the one above?